bundles/binding/org.openhab.binding.zwave/database/popp/zweather.xml

```diff
@@ -0,0 +1,98 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Product>
+	<Model>POPE005206</Model>
+	<Label lang="en">Z Weather Sensor</Label>
+	<CommandClasses>
+		<Class>
+			<id>0x20</id>
+		</Class>
+		<Class>
+			<id>0x31</id>
+		</Class>
+		<Class>
+			<id>0x32</id>
+		</Class>
+		<Class>
+			<id>0x59</id>
+		</Class>
+		<Class>
+			<id>0x5e</id>
+		</Class>
+		<Class>
+			<id>0x70</id>
+		</Class>
+		<Class>
+			<id>0x72</id>
+		</Class>
+		<Class>
+			<id>0x73</id>
+		</Class>
+		<Class>
+			<id>0x80</id>
+		</Class>
+		<Class>
+			<id>0x84</id>
+		</Class>
+		<Class>
+			<id>0x85</id>
+		</Class>
+		<Class>
+			<id>0x86</id>
+		</Class>
+		<Class>
+			<id>0x8e</id>
+		</Class>
+		<Class>
+			<id>0x8f</id>
+		</Class>
+		<Class>
+			<id>0x9b</id>
+		</Class>
+		<Class>
+			<id>0x89</id>
+		</Class>
+		<Class>
+			<id>0x60</id>
+		</Class>
+		<Class>
+			<id>0x5a</id>
+		</Class>
+	</CommandClasses>
+	<Associations>
+		<Group>
+			<Index>1</Index>
+			<Maximum>1</Maximum>
+			<Label lang="en">Life Line</Label>
+			<Help>Press the button 3 times within 1.5 seconds to add Z-Weather to
```

Comment:
Lol I guess they wouldn't be looking at the habmin help for a device if it
wasn't already included would they. Yep, sorry that was a bone head move on
my part. Will remove it.
On Jul 6, 2015 3:20 PM, "Chris Jackson" <notifications@github.com> wrote:

> In bundles/binding/org.openhab.binding.zwave/database/popp/zweather.xml
> <https://github.com/openhab/openhab/pull/2856#discussion_r33977441>:
>
> > +		<Class>
> > +			<id>0x89</id>
> > +		</Class>
> > +		<Class>
> > +			<id>0x60</id>
> > +		</Class>
> > +		<Class>
> > +			<id>0x5a</id>
> > +		</Class>
> > +	</CommandClasses>
> > +	<Associations>
> > +		<Group>
> > +			<Index>1</Index>
> > +			<Maximum>1</Maximum>
> > +			<Label lang="en">Life Line</Label>
> > +			<Help>Press the button 3 times within 1.5 seconds to add Z-Weather to
>
> Why did you put this in here? To me, this looks like either inclusion
> information, or wakeup info - but I'm not sure it has anything to do with
> association groups? (or does it?)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openhab/openhab/pull/2856/files#r33977441>.
>